modules/ROOT/pages/_partials/push/registering_device.adoc

```diff
@@ -104,6 +105,28 @@ NOTE: Optionally, you can pass the parameters below to the register method
 }
 ----
 --
+// end::excludeDownstream[]
+REST::
```

Comment:
Up to discuss if we want to add new REST for registration or document Android/IOS. 
Adding REST is kinda huge change and it will bring inconsistence.

Reply:
@secondsun WDYT? 
Maybe we can add some separate chapter?